.github/workflows/django-multitenant-tests.yml

```diff
@@ -64,6 +66,18 @@ jobs:
         --network=host \
         -e PYTHON_VERSION -e DJANGO_VERSION \
         citusdata/python-all-versions
+        docker-compose --project-name django-multitenant down
+    - name: Test GeoDjango MT
+      run: |
+        docker-compose --project-name django-multitenant up -d || { docker-compose logs && false ; }
+        echo "Running tests for python $PYTHON_VERSION, django $DJANGO_VERSION"
+        docker run \
+        --entrypoint /build/scripts/test-geodjango-mt.sh \
+        -v $(pwd):/build \
+        --network=host \
+        -e PYTHON_VERSION -e DJANGO_VERSION \
+        -e CODE_COV_APPEND="--cov-append" \
```

Comment:
Since PostGIS and Citus have seperate tests and executions, we will need seperate coverage reports
Codecov has flags feature to handle this case
https://docs.codecov.com/docs/flags
Could you set the flag to see the PostGIS coverage seperately